Update Connect-ExchangeOnline.md (#9829)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Connect-ExchangeOnline.md

After you successfully authenticate, the cmdlets in this session are mapped to the customer organization, and all operations in this session are done on the customer organization.

> Note - Please use an .onmicrosoft.com domain for the parameter value. Not doing so may result in permission related issues when you run commands in the app context. 

Reviewer: ```suggestion
**Note**: Use an .onmicrosoft.com domain for the parameter value. Not doing so might result in permission-related issues when you run commands in the app context. 
```
